addons/flashdrive/functions/fnc_compileDevice.sqf

```diff
@@ -0,0 +1,36 @@
+/**
+ * Compiles the interface from config for an object's class.
```

Comment:
interface?

Reply:
Forget this point. With the word "interface" I thought of an user interface but you meant USB interface.